Formula/qmmp.rb

```diff
@@ -1,14 +1,13 @@
 class Qmmp < Formula
   desc "Qt-based Multimedia Player"
   homepage "https://qmmp.ylsoftware.com/"
-  url "https://downloads.sourceforge.net/project/qmmp-dev/qmmp/1.5/qmmp-1.5.1.tar.bz2"
-  sha256 "f3dc676039b5f190e6a87377a6b2bd2bcca122d1659b5f22668c7a284bb91f43"
+  url "https://qmmp.ylsoftware.com/files/qmmp/2.0/qmmp-2.0.0.tar.bz2"
+  sha256 "c631d69c8bfcd77746bb94e2fc4cb7186d16cd29598de08d9771a45c212c6519"
   license "GPL-2.0-or-later"
-  head "https://svn.code.sf.net/p/qmmp-dev/code/branches/qmmp-1.5/"
 
   livecheck do
-    url :stable
-    regex(%r{url=.*?/qmmp[._-]v?(\d+(?:\.\d+)+)\.t}i)
+    url "https://qmmp.ylsoftware.com/downloads.php"
+    regex(/[0-9]+\.[0-9]+\.[0-9]+/i)
```

Comment:
```suggestion
    regex(/href=.*?qmmp[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

Since this is an HTML page, the standard for this type of check is to identify versions from archive filenames in `href` attributes (i.e., URLs to the archive files we use in the formula's `stable` URL). This anchors the regex to a specific context and ensures that we only match appropriate versions (e.g., not a version for something else on the same page or from a different file).

It's also worth noting that we use `\d` instead of `[0-9]` in `livecheck` block regexes, for the sake of brevity. For matching versions with a format like `1.2.3`, our standard regex snippet is `v?(\d+(?:\.\d+)+)`. This matches versions with two or more numeric parts, so it won't fail if there's ever a version with only two parts or more than three parts. As in the suggestion above, this snippet is part of a larger regex (anchored to a specific context) and isn't used by itself.

If you haven't already, it may help to read through the [`brew livecheck` documentation](https://docs.brew.sh/Brew-Livecheck). `livecheck` blocks in other formulae can also sometimes be a helpful starting point but they're not all representative examples.

I'll let someone who's more knowledgeable review this formula but I at least wanted to address the `livecheck` block, since that's my area of expertise.

Reply:
Thanks for your suggestion.